components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -30,6 +35,12 @@ export function getTeamSettingsMenu(params: { team?: Team; billingMode?: Billing
             link: [`/sso`],
         });
     }
+    if (orgGitAuthProviders) {
+        result.push({
+            title: "Git Integrations",
```

Comment:
`Git Integrations` is what we used to call it before. From product perspective it would make sense to reconsider this label. This standard task is called `Git Auth` in other platforms, so user expectations might benefit if we would align here.
This is not something we need to tackle here, just a note to follow up on this. 🙏🏻 Let's see how that works and discuss later.